2022-08-trading-orders.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The basics of trading - The orders
+subtitle: 
+slug: trading-orders
+tags: crypto, trading, strategy, automation, orders, blog
+cover: https://images.unsplash.com/photo-1642751226315-e6dc6b47fd54?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=3774&q=80
+domain: blog.octobot.online
+--- 
+
+Trading cryptocurrencies can be exciting, but also take a lot of time if you don’t know how to do it. So, there are many different types of orders you can make to buy or sell on your exchange  according to what you want. Read our introductory article and get a basic understanding of the orders available.
+
+# Market order, the easiest
+
+When you place a market order, you're telling your broker to buy or sell shares at the best available price. If you're buying, your broker will try to get the lowest possible price. If you're selling, they'll try to get the highest. 
+
+That's it! You don't have to worry about timing the market perfectly or anything like that. Just remember that if you're buying, you might not get the exact price you want, and if you're selling, you might not get the highest price possible. 
+
+There are other types of orders, but market orders are the easiest to understand and use.
+
+# Limit order, the most valuable  
+
+When it comes to trading, there are different types of orders that can be placed in order to execute a trade. Among these, the limit order is often considered the most valuable due to its ability to help traders get the best possible price for their desired asset.
+A limit order is a type of trade that gives you exact instructions on when and how much to buy or sell an asset at any time in future. Your trade will be executed either once your price is met or if there are significant changes in the value of this asset.
```

Comment:
"or if there are significant changes in the value of this asset" ? I don't understand what this means

Reply:
I don't understand too. I'll remove this part